bee-ternary/src/t2b1.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+use crate::{Btrit, RawEncoding, RawEncodingBuf, ShiftTernary, Utrit};
+use std::ops::Range;
+
+const TPB: usize = 2;
+const BAL: i8 = 4;
```

Comment:
should we explain those constants? They don't seem obvious. 🤔

Reply:
Just commented them.